docs/cep/query-guide/query.md

```diff
@@ -302,44 +302,44 @@ Windows help to retain events based on a criterion, such that the values of thos
-| [session](functions#session-window) | Retains events for each session based on session key. |
-| [batch](functions#batch-window) | Retains events of last arrived event chunk. |
-| [sort](functions#sort-window) | Retains top-k or bottom-k events based on a parameter value. |
-| [cron](functions#cron-window) | Retains events based on cron time in a tumbling/batch manner. |
-| [externalTime](functions#externalTime-window) | Retains events based on event time value passed as a parameter in a sliding manner.|
-| [externalTimeBatch](functions#externaltimebatch-window) | Retains events based on event time value passed as a parameter in a a tumbling/batch manner.|
-| [delay](functions#delay-window) | Retains events and delays the output by the given time period in a sliding manner.|
+**Core windows**
+
+Following are some core Stream windows, for more windows refer [here](functions/unique).
+
+| Core window function                                         | Description|
+|--------------------------------------------------------------|-------------|
+| [time](functions/core/time-window.md)                        | Retains events based on time in a sliding manner.|
+| [timeBatch](functions/core/timebatch-window.md)                 | Retains events based on time in a tumbling/batch manner. |
+| [length](functions/core/length-window.md)                       | Retains events based on number of events in a sliding manner. |
+| [lengthBatch](functions/core/lengthbatch-window.md)             | Retains events based on number of events in a tumbling/batch manner. |
+| [timeLength](functions/core/timelength-window.md)               | Retains events based on time and number of events in a sliding manner. |
+| [session](functions/core/session-window.md)                     | Retains events for each session based on session key. |
+| [batch](functions/core/batch.md)                                | Retains events of last arrived event chunk. |
+| [sort](functions/core/sort-window.md)                           | Retains top-k or bottom-k events based on a parameter value. |
+| [cron](functions/core/cron-window.md)                           | Retains events based on cron time in a tumbling/batch manner. |
+| [externalTime](functions/core/externalTime-window.md)           | Retains events based on event time value passed as a parameter in a sliding manner.|
+| [externalTimeBatch](functions/core/externaltimebatch-window.md) | Retains events based on event time value passed as a parameter in a a tumbling/batch manner.|
+| [delay](functions/core/delay-window.md)                         | Retains events and delays the output by the given time period in a sliding manner.|
 
 **Example 1**
 
 Query to find out the maximum temperature out of the **last 10 events**, using the window of `length` 10 and `max()` aggregation function, from the `TempStream` stream and insert the results into the `MaxTempStream` stream.
```

Comment:
```suggestion
Query to find out the maximum temperature out of the last 10 events, using the window of `length` 10 and `max()` aggregation function, from the `TempStream` stream and insert the results into the `MaxTempStream` stream.
```

Reply:
In general, we do not use bold to emphasize text. It is mostly for indicating something in the UI that should be clicked.